modules/health_quest/spec/services/user_service_spec.rb

```diff
@@ -0,0 +1,140 @@
+      '1Y5MDA2ODEifSwidXNlclR5cGUiOiJWRVRFUkFOIiwidmFtZi5hdXRoLnJvbGVzIjpbInZldGVyYW4iXSwicmlnaHRPZkFjY2Vzc0FjY2Vwd' \
+      'GVkIjp0cnVlLCJleHAiOjE1NzA3MzIxOTYsImp0aSI6ImViZmM5NWVmNWYzYTQxYTdiMTVlNDMyZmU0N2U5ODY0IiwibG9hIjoyfQ.HD2xgV' \
+      'YoCmF87XLlgawiCvddtkhQ0mOj7T00kh02ygY8cQhoYiylH9DaQRiFg-ymsf0xA-BHP4JqrDXLKho7wTJceRBfeYRysUSa0bbRVDPPeEuQF0' \
+      'f96DCTsL_t6ZRJB72fL4yK-Z5jovGVD8yYX6Fg4j9IJhGN2ibwJwjS6bS4I7quhm_29SjRNtjgPlvM87Lz9xg3KDHjcHBfthOvhcsnvwcsQn' \
+      'VKyvyM4ujy7nUqTF8qyJdFflDLC1F3KbY0W5IcJwR1R226Jp7K8tsfmWaZOWWD_1BITQBbdl-0jfgWcdpxpv67WBAkv3Lw9DN5dplOuan5Dq' \
+      'N-ZTMun8ub0A'
+  end
+
+  let(:refresh_token) do
+    'eyJhbGciOiJSUzUxMiJ9.eyJhdXRoZW50aWNhdGVkIjp0cnVlLCJzdWIiOiIxMDEyODQ1MzMxVjE1MzA0MyIsImF1dGhlbnRpY2F0aW9uQXV' \
+      '0aG9yaXR5IjoiZ292LnZhLnZhb3MiLCJpZFR5cGUiOiJJQ04iLCJpc3MiOiJnb3YudmEudmFtZi51c2Vyc2VydmljZS52MSIsInZhbWYuYXV0' \
+      'aC5yZXNvdXJjZXMiOlsiXi4qKFwvKT9zaXRlW3NdP1wvKGRmbi0pPzk4NFwvcGF0aWVudFtzXT9cLzU1MjE2MTA1MFwvYXBwb2ludG1lbnRzK' \
+      'FwvLiopPyQiLCJeLiooXC8pP3NpdGVbc10_XC8oZGZuLSk_OTgzXC9wYXRpZW50W3NdP1wvNzIxNjY5MVwvYXBwb2ludG1lbnRzKFwvLiopPy' \
+      'QiLCJeLiooXC8pP3BhdGllbnRbc10_XC8oSUNOXC8pPzEwMTI4NDUzMzFWMTUzMDQzKFwvLiopPyQiXSwidmVyc2lvbiI6Mi4xLCJ2aXN0YUl' \
+      'kcyI6W3sicGF0aWVudElkIjoiNTUyMTYxMDUwIiwic2l0ZUlkIjoiOTg0In0seyJwYXRpZW50SWQiOiI3MjE2NjkxIiwic2l0ZUlkIjoiOTgz' \
+      'In1dLCJzdGFmZkRpc2NsYWltZXJBY2NlcHRlZCI6dHJ1ZSwibmJmIjoxNTkyNTIyODY1LCJzc3QiOjE1OTI1MjI5NDksInBhdGllbnQiOnsia' \
+      'WNuIjoiMTAxMjg0NTMzMVYxNTMwNDMifSwicmlnaHRPZkFjY2Vzc0FjY2VwdGVkIjp0cnVlLCJ2YW1mLmF1dGgucm9sZXMiOlsidmV0ZXJhbi' \
+      'JdLCJleHAiOjE1OTI1MjM5NDUsImp0aSI6IjEzM2JhZjVkLWZjNjItNGY0ZS04YTkxLWY2YjE2MmQzODk0ZiIsImxvYSI6Mn0.Q0RuJojxa1X' \
+      'asQGFrMD4TZJqMGPJL2Y-2S0i_PEzt7ODQXSah4MnJAbT8r48D91yvCBR8x4tMArRaO-1_SpxFKOMT-ysXqcYV-LeZyuewiuNO6c8gfVUwsOs' \
+      'SF0FYY2RqV33OHEdSYQu_wMCVf-1mV5nKURQcNtaOH1vw42zruu6JUooEqSUgzLXeMmjrZVMQyeOBVHsNCV-BEIUWyPta1HcnLr-z0hyASS1Z' \
+      'VpEqDzdWOaWmAgVJGj0ctyYZQG-kbLs_t36zkN8XK3HEN1-Gjy2WLGjLLlHQbG3AFHih0pyiM2NsUSJWH0_r_S2wF4h-GtXeIPckS2JfBZ0F5' \
+      'HM1A'
+  end
+
+  before do
+    allow(HealthQuest::Configuration.instance).to receive(:rsa_key).and_return(rsa_key)
+    time = Time.utc(2019, 10, 10, 18, 14, 56)
+    Timecop.freeze(time)
+  end
+
+  after { Timecop.return }
+
+  describe '#session' do
+    describe '#session' do
+      let(:response) { double('response', body: token) }
+
+      context 'with a 200 response' do
+        it 'returns the session token' do
+          VCR.use_cassette('health_quest/users/post_session') do
+            session_token = subject.session(user)
+            expect(session_token).to be_a(String)
+          end
+        end
+
+        it 'makes a call out to the the health_quest user service once' do
+          VCR.use_cassette('health_quest/users/post_session') do
+            # LJG NOTE: this code was copied from VAOS and it does not fail rubocop, not sure why
```

Comment:
there's an exception here https://github.com/department-of-veterans-affairs/vets-api/blob/960e5439eda33d813122d02ca934e8ce07538e42/.rubocop.yml#L317 but I think putting the exception in the code is better.  Remove your comment here and maybe add an issue to refactor this later.

Reply:
ok, I will leave the rubocop disable and remove my comment. Thanks. I started a different PR by the way

Reply:
Which PR do you want to work with moving forward?  Let's close the one we're not using

Reply:
I closed the other PR